### PR TITLE
chore(messaging-documentation) - Add GH page for Message Router

### PR DESCRIPTION
**Pull request by lilla28**
Changes:
- Added GH page for Message Router

### CODE REVIEW

**Review by psmulovics — APPROVED** [review]

**Comment by kruplm** [comment]
FYI I'm checking a local build with the changes at the moment.

**Comment by kruplm** [comment]
@psmulovics @lilla28 This is looking good to me based on the local build:
docu-1
docu-2
docu-3

@mimiflynn feel free to verify as well :)

File: site/content/documentation/v0.1.0-alpha.4/message-router.mdx [hunk 1]
```diff
@@ -0,0 +1,220 @@
+---
+order: 2
+title: Message Router API documentation
+---
+
+The Message Router is responsible for every communication that happens between processes. It is an independent pluggable module that runs under the main ComposeUI process, delivering messages between processes and optionally other devices.
+Message Router is an open-source out-of-the-box library that is fast and reliable. It supports JSON and it also uses WebSockets and has a Javascript library, so Web-based applications can be integrated. 
+So basically it is real-time solution that can be easily setup for Pub-Sub messaging.
+
+More details can be found under the [ADR-012](https://github.com/morganstanley/ComposeUI/blob/main/architecture/adr-012-message-router.md).
```
Comment (mimiflynn) [inline site/content/documentation/v0.1.0-alpha.4/message-router.mdx:10]: Any interest in having a section of the documentation site for these ADRs?

**Review by mimiflynn — APPROVED** [review]

File: site/content/documentation/v0.1.0-alpha.4/message-router.mdx (hunk 1 above)
Comment (psmulovics) [reply site/content/documentation/v0.1.0-alpha.4/message-router.mdx:10]: @mimiflynn , definitely. Where would you suggest putting them?

Comment (mimiflynn) [reply site/content/documentation/v0.1.0-alpha.4/message-router.mdx:10]: Would it makes sense to keep in the docs section or do you think a separate ADR space would be better?

Comment (psmulovics) [reply site/content/documentation/v0.1.0-alpha.4/message-router.mdx:10]: I'm far from being a designer :) but I'd put it under docs.

**Pull request merged**